lisp/ess-inf.el

```diff
@@ -1830,10 +1796,22 @@ far as possible and return -1."
     (goto-char pos)
     n))
 
+(defun ess-step (&optional skip)
+  "Move point to the end of SKIP (if given) and then step.
```

Comment:
How bout `ess-step-thing` instead? `ess-step` is too generic. Also `ess-skip-thing` sounds a bit more meaningful to my ears.

Reply:
I chose `step` to be consistent with all the functions that use it (e.g. ess-eval-*-and-step).

How about `ess-skip-and-step`? or `ess-step-maybe-skip`?